### PR TITLE
ddl: disable raw mode resource group

### LINKED ISSUE #41049: resource group ddl bug
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

set global tidb_enable_resource_control = 'on';
create resource group rg1 ru_per_sec=1000 ru_per_sec=2000;  <--- statement 1 
create resource group rg2 CPU='4000m' IO_READ_BANDWIDTH='1G' IO_WRITE_BANDWIDTH='300M'; <--- statement 2

### 2. What did you expect to see? (Required)
statement 1 failed by parser
statement 2 is blocked

### 3. What did you see instead (Required)

statement 1 and 2 succeed

### 4. What is your TiDB version? (Required)

master

### PR DESCRIPTION
**Pull request by BornChanger**
Signed-off-by: BornChanger <dawn_catcher@126.com>

### What problem does this PR solve?

Issue Number: close #41049 

Problem Summary:

### What is changed and how it works?
This patch addressed following issues:
1. Disable raw mode resource group creation
2. Fix a bug of redundant ru_per_sec option in DDL statements
`create resource group rg1 ru_per_sec=100 ru_per_sec=200` will fail with parser error

### Check List

Tests 

- [X] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -1598,36 +1598,28 @@ ResourceGroupOptionList:
...
+		$$ = append([]*ast.ResourceGroupOption{$1.(*ast.ResourceGroupOption)}, &ast.ResourceGroupOption{Tp: ast.ResourceBurstableOpiton, BoolValue: true})
 	}
+|	"BURSTABLE" DirectResourceGroupOption
+    {
+    	$$ = append([]*ast.ResourceGroupOption{$2.(*ast.ResourceGroupOption)}, &ast.ResourceGroupOption{Tp: ast.ResourceBurstableOpiton, BoolValue: true})
+    }
+|	"BURSTABLE" ',' DirectResourceGroupOption
+    {
+    	$$ = append([]*ast.ResourceGroupOption{$3.(*ast.ResourceGroupOption)}, &ast.ResourceGroupOption{Tp: ast.ResourceBurstableOpiton, BoolValue: true})
+    }
```
Comment (tiancaiamao) [inline parser/parser.y:1616]: It looks confusing... **where BURSTABLE is allowed exactly**?


```
create resource group xx  BURSTABLE RU_PER_SEC = 123 BURSTABLE , BURSTABLE RU_PER_SEC  = 456
```

Comment (BornChanger) [reply parser/parser.y]: I reworked the yacc file to make logic more readable.

**Review by tiancaiamao — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**